designs/initial.md

```diff
@@ -0,0 +1,60 @@
+## `@rest` directive
+
+### Arguments
+
+- route: path to rest endpoint. This could be a path or a full url. If a path, add to the endpoint given on link creation or from the context
```

Comment:
- The assumption I'm making here is that this is intended to be a "base" for subsequent queries to add paths on-to. Is that assumption correct?
- If I have multiple REST servers/apis I'm trying to contact, do I get multiple `@rest<foo>` directives? or is there an identifier that we can optionally pass? Aka `@rest(host: "github", path: …` vs `@restGithub(…`

Reply:
Yeah figuring out a way to easily support multiple endpoints is critical! I think we should keep the single `@rest` directive, but make the endpoint easier to change? @peggyrayzis put below the idea of a map and then you can use a key as the endpoint in the directive